Libs/Visualization/VTK/Widgets/ctkVTKColorTransferFunction.h

```diff
@@ -48,26 +48,26 @@ class CTK_VISUALIZATION_VTK_WIDGETS_EXPORT ctkVTKColorTransferFunction: public c
   
   /// Please note that controlPoint methods only works if you have at least one
   /// ControlPoint.
-  virtual ctkControlPoint* controlPoint(int index)const;
-  virtual QVariant value(qreal pos)const;
-  virtual int count()const;
-  virtual bool isDiscrete()const;
-  virtual bool isEditable()const;
+  Q_INVOKABLE virtual ctkControlPoint* controlPoint(int index)const;
+  Q_INVOKABLE virtual QVariant value(qreal pos)const;
+  Q_INVOKABLE virtual int count()const;
+  Q_INVOKABLE virtual bool isDiscrete()const;
```

Comment:
For bool accessors we usually don't define invokable but set it as Qt property instead.